server/api/users/users.test.ts

```diff
@@ -72,4 +72,96 @@ describe('auth router', () => {
+        discord_name: `${randUserName()}#${randNumber()}`,
+        bio: 'I like to put cat pictures on Caleb\'s desktop & my owner owns nerdwallet',
+        twitter_username: randUserName(),
+        linkedin_url: `https://www.linkedin.com/in/${randUserName()}/`,
+        github_username: randUserName(),
+        website: 'https://leonnoel.com/',
+      });
+
+      server.login(user);
+
+      const fieldsToUpdate = {
+        bio: 'I was updated.',
+        twitter_username: 'Updated@updated',
+        linkedin_url: 'updated',
+        github_username: 'gotUpdated',
+        website: 'https://thiswasupdated.com/',
+      };
+
+      const res = await server.exec.patch(`/api/users/${server.loggedInUser.dataValues.id}`).send(fieldsToUpdate);
+      expect(res.status).toBe(200);
+      expect(res.body).toEqual(`User of id ${server.loggedInUser.dataValues.id} updated.`);
+
+      const updatedUser = await server.exec.get(`/api/users/${server.loggedInUser.dataValues.id}`);
+      expect(updatedUser.status).toBe(200);
+      expect(updatedUser.body).toEqual({
+        id: user.id,
+        discord_user_id: user.discord_user_id,
+        discord_name: user.discord_name,
+        bio: 'I was updated.',
+        twitter_username: 'Updated@updated',
+        linkedin_url: 'updated',
+        github_username: 'gotUpdated',
+        website: 'https://thiswasupdated.com/',
+      });
+    });
+
+    it('removes markup from input fields', async () => {
+      const user = await User.create({
+        email: randEmail(),
+        discord_user_id: randNumber().toString(),
+        discord_name: `${randUserName()}#${randNumber()}`,
+        bio: 'I like to put cat pictures on Caleb\'s desktop & my owner owns nerdwallet',
+        twitter_username: randUserName(),
+        linkedin_url: `https://www.linkedin.com/in/${randUserName()}/`,
+        github_username: randUserName(),
+        website: 'https://leonnoel.com/',
+      });
+
+      server.login(user);
+
+      const fieldsToUpdate = {
+        bio: '<script>alert("hi")</script>',
+        twitter_username: 'Updated@updated',
+        linkedin_url: 'updated',
+        github_username: 'gotUpdated',
+        website: 'https://thiswasupdated.com/',
+      };
+
+      const res = await server.exec.patch(`/api/users/${server.loggedInUser.dataValues.id}`).send(fieldsToUpdate);
+      expect(res.status).toBe(200);
+      expect(res.body).toEqual(`User of id ${server.loggedInUser.dataValues.id} updated.`);
+
+      const updatedUser = await server.exec.get(`/api/users/${server.loggedInUser.dataValues.id}`);
+      expect(updatedUser.status).toBe(200);
+      expect(updatedUser.body).toEqual({
+        id: user.id,
+        discord_user_id: user.discord_user_id,
+        discord_name: user.discord_name,
+        bio: '',
+        twitter_username: 'Updated@updated',
+        linkedin_url: 'updated',
+        github_username: 'gotUpdated',
+        website: 'https://thiswasupdated.com/',
+      });
+    });
```

Comment:
i think we can actually do away with this test and instead add tests to the middleware functions!  see next comment